zkml/src/onnx_parse.rs

```diff
@@ -1,52 +1,39 @@
 use crate::{
```

Comment:
Should we just get rid of this module now ? We have everything in parser/ now i believe?

Reply:
i keep the `FloatOnnxLoader` structure in this file, which is using the actual parser to build, quantize and pad the model

Reply:
I would rather move that to the onnx parser submodule now ? wdyt ?

Reply:
Done in commit https://github.com/Lagrange-Labs/deep-prove/pull/114/commits/fa37beb0683993623225ff7b66e3fbfa99564baf